src/client/pages/ConnectPayment/index.tsx

```diff
@@ -2,16 +2,33 @@ import { SemanticEvents } from 'quepasa'
 import React from 'react'
 import Helmet from 'react-helmet-async'
 import { Mount } from 'react-lifecycle-components/dist'
+import { useHistory } from 'react-router'
 import { TopBar } from 'components/TopBar'
 import { Page } from 'components/utils/Page'
 import { SessionTokenGuard } from 'containers/SessionTokenGuard'
 import { LanguagePicker } from 'pages/Embark/LanguagePicker'
 import { useTextKeys } from 'utils/textKeys'
 import { getUtmParamsFromCookie, TrackAction } from 'utils/tracking/tracking'
+import { PhoneNumber } from 'components/PhoneNumber/PhoneNumber'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { pushToGTMDataLayer } from 'utils/tracking/gtm'
 import { ConnectPaymentPage } from './sections/ConnectPayment'
 
 export const ConnectPayment: React.FC = () => {
   const textKeys = useTextKeys()
+  const currentLocale = useCurrentLocale()
+
+  const history = useHistory()
+
+  const onClickPhoneTracking = (path: string, status: 'opened' | 'closed') => {
```

Comment:
I would consider naming this:

```suggestion
  const handleClickPhoneNumber = (path: string, status: 'opened' | 'closed') => {
```

Reply:
Great, I'll fix!